src/local.rs

```diff
@@ -31,3 +32,12 @@ impl<'sc, T> DerefMut for Local<'sc, T> {
     unsafe { self.0.as_mut() }
   }
 }
+
+impl<'sc, T> From<Local<'sc, T>> for Local<'sc, Value>
```

Comment:
This is certainly an improvement - but please add a TODO to make it possible for targets other than `Local<Value>`.. For example `Local<String>` should be able to be down cast to `Local<Name>`.